src/smc-webapp/r_misc/A.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from "react";
 
 interface AProps {
   href: string;
-  children: string;
+  children: React.ReactNode;
```

Comment:
This should be string | React.ReactNode, since you pass in a bunch of strings.  I'm surprised this even compiles...